--- origsrc/Togl2.0/Makefile.in	2008-04-21 18:42:53.000000000 -0500
+++ src/Togl2.0/Makefile.in	2008-07-02 21:51:02.968750000 -0500
@@ -59,7 +59,7 @@
 # This is a list of public header files to be installed, if any.
 #========================================================================
 
-PKG_HEADERS	= @PKG_HEADERS@ togl_ws.h
+PKG_HEADERS	= @PKG_HEADERS@
 
 #========================================================================
 # "PKG_LIB_FILE" refers to the library (dynamic or static as per
@@ -290,6 +290,7 @@
 	    echo "Installing $(srcdir)/$$i" ; \
 	    $(INSTALL_DATA) $(srcdir)/$$i $(DESTDIR)$(includedir) ; \
 	done
+	$(INSTALL_DATA) togl_ws.h $(DESTDIR)$(includedir)
 
 #========================================================================
 # Install documentation.  Unix manpages should go in the $(mandir)
@@ -509,11 +510,6 @@
 	@if test "x$(SHARED_BUILD)" = "x1"; then \
 	    echo " Install pkgIndex.tcl $(DESTDIR)$(pkglibdir)"; \
 	    $(INSTALL_DATA) pkgIndex.tcl $(DESTDIR)$(pkglibdir); \
-	    echo " Install LICENSE $(DESTDIR)$(pkglibdir)"; \
-	    $(INSTALL_DATA) LICENSE $(DESTDIR)$(pkglibdir); \
-	else \
-	    echo " Install LICENSE.togl $(DESTDIR)$(libdir)"; \
-	    $(INSTALL_DATA) LICENSE $(DESTDIR)$(libdir)/LICENSE.togl; \
 	fi
 
 #========================================================================
